modules/core/src/v2/coins/eth2.ts

```diff
@@ -0,0 +1,392 @@
+/**
+ * @prettier
+ */
+import * as Bluebird from 'bluebird';
+import * as debugLib from 'debug';
+import * as _ from 'lodash';
+import * as request from 'superagent';
+import { Eth2 as Eth2AccountLib } from '@bitgo/account-lib';
+
+import {
+  BaseCoin,
+  FeeEstimateOptions,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+  HalfSignedTransaction as BaseHalfSignedTransaction,
+} from '../baseCoin';
+import { BitGo } from '../../bitgo';
+import { NodeCallback } from '../types';
+import * as common from '../../common';
+import { EthereumLibraryUnavailableError } from '../../errors';
+
+const co = Bluebird.coroutine;
+const debug = debugLib('bitgo:v2:eth');
+
+export const optionalDeps = {
+  get ethUtil() {
```

Comment:
It looks like your only usage of `ethUtil` is for the `BigNumber` package inside... Let's just remove ethUtil and optionalDeps and just use bignumber on its own